docker/quick-start/bacalhau-entrypoint.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+SUDO='' # detect if not root...
+
+if [[ -z "${IPFS_CONNECT:-}"  ]]; then
+  # if bacalhau-ipfs container isn't running, start one
+  if ! docker inspect bacalhau-ipfs > /dev/null ; then
+    # TODO: can we put the desired version into the bacalhau binary?
+    export IPFS_VERSION=$(wget -q -O - https://raw.githubusercontent.com/filecoin-project/bacalhau/main/ops/terraform/production.tfvars | grep --color=never ipfs_version | awk -F'"' '{print $2}')
+    #ignoring the repo version - they're all too old atm
+    export IPFS_VERSION=latest
+    # run IPFS (this should eventually be run in the bacalhau entrypoint)
+    # TODO: run container as readonly fs
+
+    echo "Starting the backalhau-ipfs ${IPFS_VERSION} container"
+    docker run -d \
```

Comment:
This doesn't feel right. Starting a dependent container in the entrypoint script of another container? Surely it's easier to start it separately (or use docker compose or something?)

Alternatively, is it possible to bundle the kubo binary at build time, then run a forked process to run kubo?

(Again, I'll check the notes in a second)

Reply:
For cleanup reasons (and because i was stuffing with different versions), I wanted to have kubo in a separate container, with separate volumes - I'm going to head further down that route as that fits the compose route.

nicely, the making of `bacalhauproject/bacalhau:<commit sha>` and `bacalhauproject/bacalhau:v0.17.0` for use in a compose file, isn't incompatible with using that same image as `docker run --dir -v dockersockt bacalhauproject/bacalhau:<commit sha> --devstack`